PlatformUI/EventListener/PlatformUIListener.php

```diff
@@ -19,10 +19,6 @@ protected function isPlatformUIRequest(Request $request)
             return false;
         }
 
-        if (!$request->headers->has('X-PJAX') || !$request->isXmlHttpRequest()) {
-            return false;
-        }
-
-        return true;
+        return $request->headers->has('X-AJAX-Update') || $request->isXmlHttpRequest();
```

Comment:
Is this right?

Not every AJAX request is Platform UI 2.x request :)

Will this work for Platform UI 1.x?